app/views/curriculums/_certification_fields.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="card-panel">
+  <div class='nested-fields'>
+    <div class="row">
+      <div class="col s12">
+        <%= f.label :Nombre_del_curso_o_certificacion %>
+        <%= f.text_field :certification_name %>
+      </div>
+    </div>
+    
+    <div class="row">
+      <div class="col s12 l10">
+        <%= f.label :Organizacion_o_Universidad %>
+        <%= f.text_field :emitting_organization %>
+      </div>
+      <div class="col s12 l2">
+        <%= f.label :Fecha %>
+        <%= f.date_field :date_emitted %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col s12">
+        <%= f.label :Descripcion_breve %>
+        <%= f.text_area :brief_description %>
```

Comment:
Can you add the css class to style the text are with materialize. There are some examples in our codebase.